decidim-core/lib/tasks/decidim_webpacker_tasks.rake

```diff
@@ -30,6 +30,19 @@ namespace :decidim do
       # Remove the webpacker dependencies as they come through Decidim dependencies.
       # This ensures we can control their versions from Decidim dependencies to avoid version conflicts.
       webpacker_packages = %w(
+        @babel/core
```

Comment:
We temporarily remove what shakapacker has to offer, so we could tackle in a separate PR ( that would make use of new babel & friends).